exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/http/trace/OtlpHttpSpanExporter.java

```diff
@@ -22,12 +26,13 @@
 @ThreadSafe
 public final class OtlpHttpSpanExporter implements SpanExporter {
 
-  private final HttpExporterBuilder<TraceRequestMarshaler> builder;
-  private final HttpExporter<TraceRequestMarshaler> delegate;
+  private static final boolean LOW_ALLOCATION_MODE = true;
```

Comment:
This flag is there just to enable the low allocation mode so that tests would run using it. Eventually I'm sure this will be replaced by something else, be it memory mode or some other flag.